temporal/api/cloud/nexus/v1/message.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package temporal.api.cloud.nexus.v1;
+
+option go_package = "go.temporal.io/api/cloud/nexus/v1;nexus";
+
+import "google/protobuf/timestamp.proto";
+
+message IncomingServiceSpec {
+    // The name of the incoming service. Must be unique within an account
+    // The name must match `[a-zA-Z_][a-zA-Z0-9_]*`.
+    string name = 1;
+
+    // The namespace to route requests to
+    string namespace = 2;
+
+    // The task queue to route requests to
+    string task_queue = 3;
+}
+
+// A binding from a service name to namespace and task queue for dispatching incoming Nexus requests.
+message IncomingService {
+    // The id of the service. This is generated by the server and is immutable
+    string id = 1;
+
+    // The current version of the incoming service specification
+    // The next update operation must include this version
+    string resource_version = 2;
```

Comment:
resource_version is control plane's flavor of "version/ETag", is a string and is consistent with other resources. This is another difference between this and the core OSS API

Reply:
That's fine IMHO.